src/Flash.tsx

```diff
@@ -122,7 +125,7 @@ export const Flash = ({
 
   return (
     <div className={cls} style={style}>
-      <span className={`${stylePrefix}__value`}>{valueFormatter(value)}</span>
+      {render(value, valueFormatter)}
```

Comment:
Should we keep the `<span />` though? This removes one of the available styling hooks... I suspect the styled examples in Storybook might be broken with this removed. Take a look and see, I suspect we need to keep this.

Reply:
Good idea passing the formatter in.

Reply:
Can we add, as part of this PR, tests for this new functionality? I know we don't have any real tests for this component at all, which we should do in another PR, but it would be great to get started on that now.